data_import/views.py

```diff
@@ -75,3 +86,39 @@ def get(self, request, *args, **kwargs):
         return HttpResponseForbidden(
             "<h1>You are not authorized to view this file.</h1>"
         )
+
+
+class NewDataFileAccessLogView(NeverCacheMixin, ListAPIView):
+    """
+    Return the list of public data files.
```

Comment:
`Custom API endpoint returning logs of file access requests for OHLOG_PROJECT_ID`